var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -87,31 +85,39 @@ class PyNumpy(PythonPackage):
     version("1.14.6", sha256="1250edf6f6c43e1d7823f0967416bc18258bb271dc536298eb0ea00a9e45b80a")
     version("1.14.5", sha256="a4a433b3a264dbc9aa9c7c241e87c0358a503ea6394f8737df1683c7c9a102ac")
 
-    variant("blas", default=True, description="Build with BLAS support")
-    variant("lapack", default=True, description="Build with LAPACK support")
```

Comment:
Is anyone actually trying to build numpy without BLAS/LAPACK support?

Reply:
A surprising number of people do - often when they don't care much about performance and find it easier to not deal with a non-PyPI dependency.

Reply:
Hmm, is that something we want to support? Or is Spack good enough at building non-PyPI deps that everyone will just use OpenBLAS/MKL? Trying to keep the package as simple as possible if we can.

Reply:
No, I don't think it's a thing that Spack should support. Conda-forge and Linux distros don't either. There's no good reason to accept a huge performance penalty here.